src/core/include/openvino/op/range.hpp

```diff
@@ -39,11 +39,18 @@ class OPENVINO_API Range : public Op {
     void set_output_type(element::Type output_type) {
         m_output_type = output_type;
     }
+
     // Overload collision with method on Node
     using Node::set_output_type;
```

Comment:
Can we create a similar getter for the `m_output_type` (that will return a constant reference) so that we can use it inside the `shape_infer` function and not make this function a friend (which will allow us to set a default value for the data_constant)?

Reply:
OK